server/build.gradle

```
@@ -40,6 +40,7 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-compiler:$kotlinVersion"
     implementation "org.jetbrains.kotlin:kotlin-scripting-compiler:$kotlinVersion"
     implementation "org.jetbrains.kotlin:kotlin-scripting-jvm-host-unshaded:$kotlinVersion"
+    implementation 'org.jetbrains.kotlin:kotlin-sam-with-receiver-compiler-plugin'
```

Should we pin this version too? Or does the dependency resolver do that automatically?

```suggestion
    implementation "org.jetbrains.kotlin:kotlin-sam-with-receiver-compiler-plugin:$kotlinVersion"
```